examples/patchMultipleObjects.md

```diff
@@ -216,3 +223,32 @@ diff $DEMO_HOME/out_actual.yaml $DEMO_HOME/out_expected.yaml
 
 To see how to do this with JSON patches,
 try the [JSON patch] demo.
+
+### Limitations
+  
+Kustomize `patches` may be applied to multiple resources using
+target-specifiers, but a single patch cannot apply to multiple
+parts of the same resource's configuration. This is true for
+both strategic merge patches and json patches.
+  
+For example, there is presently no way to express the operation
+"append a new entry to the `env` list for every container in a
```

Comment:
Please note that we would like to have the replacements transformer support this, if someone has bandwidth to contribute it, tracked here: https://github.com/kubernetes-sigs/kustomize/issues/4561

It currently works as long as the field in question is already present (i.e. to overwrite the value). See the wildcard format toward the end of this section: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/replacements/#field-path-format. Since this doc is about how to patch multiple objects, I think discussion of the replacements transformer approach would be a valuable addition to it.